docs/source/cdx_api.rst

```diff
@@ -7,17 +7,127 @@ Python API to Internet Archive Wayback Machine
 Search for historical snapshots of a URL. Download metadata about the snapshots
 and/or the snapshot content itself.
 
-We implement Python clients for the CDX and Memento APIs provided by Wayback
```

Comment:
Note that I didn't _delete_ this; I just moved it way down the pay and rephrased a bit.